### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by humzahasan**
PR Description :

Rules Modal and Landing page added

[Check Me](https://github.com/humzahasan/Quizified/pull/4/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&file-filters%5B%5D=.jsx&show-viewed-files=true)

### CODE REVIEW

File: src/pages/Landing/Landing.jsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+              )}
+              <div className="card-media">
+                <img src={Movieimg} alt="Movie Category" className="card-img" />
+                <p className="card-img-overlay overlay-hover"> TV & Movies</p>
+              </div>
+            </div>
+          </div>
+          {(selectedFood || selectedMovie) && (
+            <div className="landing-info">
+              {/* <p className="md-title">Selected Category : {category} </p> */}
```
Comment (itsmebipul00) [inline src/pages/Landing/Landing.jsx:76]: remove commented code

File: src/pages/Landing/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                type="text"
+                placeholder="Enter Your Name"
+                className="input-box"
+                // value={username}
+                // onChange={(e) => setUsername(e.target.value)}
+              />
+              <button
+                // disabled={!username}
+                // className={username ? "btn btn-primary" : "btn btn-disabled"}
+                onClick={sendQuizInfo}
```
Comment (itsmebipul00) [inline src/pages/Landing/Landing.jsx:87]: here   too

File: src/pages/Landing/Landing.jsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+              Leaderboard
+            </Link>
+          </ul>
+        </footer>
+      </div>
+      <Rules
+        handleClose={backToLanding}
+        showRules={showRules}
+        // category={category}
+        // username={username}
```
Comment (itsmebipul00) [inline src/pages/Landing/Landing.jsx:111]: here too

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

**Review by tonymike99 — APPROVED** [review]

File: src/pages/Landing/Landing.jsx (hunk 1 above)
Comment (humzahasan) [reply src/pages/Landing/Landing.jsx:76]: Will be using this in next PR

**Pull request merged**